tests/unit/amo/components/TestAddon.js

```diff
@@ -1566,4 +1568,71 @@ describe(__filename, () => {
       expect(installButton).toHaveProp('uninstall', uninstall);
     });
   });
+  // non-public add-ons require an account listed as a developer of the add-on or admin rights
```

Comment:
hey @championshuttler, can you capitalize the first letter Non-public.. and end with a period?

Reply:
:+1: